samples/subsys/fs/README.rst

```diff
@@ -0,0 +1,29 @@
+.. _fat_fs:
+
+FAT Filesystem Sample Application
+###################################
+
+Overview
+********
+
+This sample app demonstrates use of the filesystem API and uses the Fat file
+system driver to mount an SDHC card connected over SPI bus.
+
+Requirements
+************
+
+This project requires a SDHC or microSD card formatted with fat filesystem.
+Please see documentation for the SDHC driver for any current limitations.
+
+Building and Running
+********************
+
+This sample can currently be built for ``nrf52840_blip`` board. It requires
+both the ``nrf52840_blip.overlay`` and the ``dts_fixup.h`` for nrf52840_blip
+to work:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/subsys/fs
+   :board: nrf52840_blip
+   :goals: build
+   :compact:
```

Comment:
After you build this sample, how do you run it?  Please add that and mention what happens when you do run it: what does the sample do?

Reply:
added details